src/NServiceBus.AcceptanceTests/Sagas/When_correlated_property_value_is_changed.cs

```diff
@@ -21,10 +21,9 @@ await Scenario.Define<Context>()
                         {
                             DataId = Guid.NewGuid()
                         })))
-                    .Done(c => c.FailedMessages.Any())
+                    .Done(c => c.ModifiedCorrelationProperty)
```

Comment:
Does this change actually have the desired outcome? Are we now racing to shutdown before actually finishing processing the saga and getting the exception thrown?